README.md

```diff
@@ -29,7 +29,7 @@ Deploy and log in with username *admin* and the password that's set up during de
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
  * PHP <b>7.2</b> or higher
- * [Composer](https://getcomposer.org/)
+ * [Composer](https://getcomposer.org/) <b>1.4.2</b> or higher
```

Comment:
how did you arrive at this version? It seems oddly specific for a minimum version.

Reply:
I made an issue (linked above) and Tom said this version works for him. I figured we'd use the oldest one reported.

Reply:
can we just say > 1.4? 1.4.2 makes it sound like we have some known bug with 1.4.1